src/file_analysis/CMakeLists.txt

```diff
@@ -1,23 +1,17 @@
-include(ZeekSubdir)
-
-include_directories(BEFORE
-                    ${CMAKE_CURRENT_SOURCE_DIR}
-                    ${CMAKE_CURRENT_BINARY_DIR}
+zeek_add_subdir_library(
+    file_analysis
+    INTERNAL_DEPENDENCIES ${BIF_BUILD_TARGET}
```

Comment:
This library depends on `BIF_BUILD_TARGET` but the broker one depended on `bif_deps`. What's the difference there?

Reply:
Thanks for pointing those out. Seems like they've been a leftover and I just didn't clean them up.